eval/run_over_csv.py

```diff
@@ -0,0 +1,39 @@
+from ice.recipe import Recipe, recipe
+import pandas as pd
+from ice.contrib.ought_shared.utils import script_run_time
+from ice.contrib.anthropic.anthropic_qa import anthropic_qa_from_kwargs
+from ice.utils import map_async
+# from importlib import import_module
+
+FILEPATH = "data/in_app_qa_results/qa_eval_output.csv"
```

Comment:
@lslunis I'd like to pass in the data filepath and the recipe as command line args, but I don't know the best way to do that. Should we pair?

Reply:
Or do you want to try to just tell me how to do it, async?

Reply:
You need to add another defopt parser [here](https://github.com/oughtinc/ice/blob/main/ice/recipe.py#L157), where the key is the dataframe type and the value is a str -> df function. Happy to pair if you have any issues.